scripts/cnn_variant_wdl/cnn_score_variants.wdl

```diff
@@ -0,0 +1,331 @@
+                mem_gb = cnn_task_mem_gb,
+                cpu = cnn_task_cpu
+        }
+    }
+
+    call MergeVCFs as MergeVCF_CNN {
+        input: 
+            input_vcfs = CNNScoreVariants.cnn_annotated_vcf,
+            output_vcf_name = output_prefix,
+            preemptible_attempts = preemptible_attempts,
+            gatk_override = gatk_override,
+            gatk_docker = gatk_docker
+    }
+
+    call FilterVariantTranches {
+        input:
+            input_vcf = MergeVCF_CNN.merged_vcf,
+            input_vcf_index = MergeVCF_CNN.merged_vcf_index,
+            resource_fofn = resource_fofn,
+            resource_fofn_index = resource_fofn_index,
+            output_prefix = output_prefix,
+            tranches = tranches,
+            info_key = info_key,
+            gatk_override = gatk_override,
+            preemptible_attempts = preemptible_attempts,
+            gatk_docker = gatk_docker
+    }
+
+    output {
+        FilterVariantTranches.*
+    }
+}
+
+task CNNScoreVariants {
+    File input_vcf
+    File input_vcf_index
+    File reference_fasta
+    File reference_dict
+    File reference_fasta_index
+    String output_prefix
+    File? bam_file
+    File? bam_file_index
+    File? architecture_json
+    File? architecture_hd5
+    Int? inference_batch_size
+    Int? transfer_batch_size
+    Int? intra_op_threads
+    Int? inter_op_threads
+    String? tensor_type
+
+    File interval_list
+    File? gatk_override
+
+    # Runtime parameters
+    Int? mem_gb
+    String gatk_docker
+    Int? preemptible_attempts
+    Int? disk_space_gb
+    Int? cpu 
+
+    # You may have to change the following two parameter values depending on the task requirements
+    Int default_ram_mb = 6000
+    Int default_disk_space_gb = 100
+
+    # Mem is in units of GB but our command and memory runtime values are in MB
+    Int machine_mem = if defined(mem_gb) then mem_gb *1000 else default_ram_mb
+    Int command_mem = machine_mem / 2 # was machine_mem - 1000
+
+command <<<
+    
+        set -e
+        export GATK_LOCAL_JAR=${default="/root/gatk.jar" gatk_override}
+
+        gatk --java-options "-Xmx${command_mem}m" \
+        CNNScoreVariants \
+        ${"-I " + bam_file} \
+        -R ${reference_fasta} \
+        -V ${input_vcf} \
+        -O ${output_prefix}_cnn_annotated.vcf.gz \
+        -L ${interval_list} \
+        ${"--architecture " + architecture_json} \
+        ${"--tensor-type " + tensor_type} \
+        ${"--inference-batch-size " + inference_batch_size} \
+        ${"--transfer-batch-size " + transfer_batch_size} \
+        ${"--intra-op-threads " + intra_op_threads} \
+        ${"--inter-op-threads " + inter_op_threads}
+
+>>>
+
+  runtime {
+    docker: "${gatk_docker}"
+    memory: machine_mem + " MB"
+    disks: "local-disk " + select_first([disk_space_gb, default_disk_space_gb]) + " HDD"
+    preemptible: select_first([preemptible_attempts, 3])
+    cpu: select_first([cpu, 1])
+    zones: "us-central1-b"
```

Comment:
Looking deeper into this it seems all of google cloud will guarantee AVX the problems we ran into earlier could have been because AVX2 wasn't available, or it may have been a library was missing from the docker.